src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -442,13 +442,18 @@ private static void UpdateGasUp(long refund, ref long gasAvailable)
             gasAvailable += refund;
         }
         
-        private static bool ChargeAccountAccessGas(ref long gasAvailable, EvmState vmState, Address address, IReleaseSpec spec, bool chargeForWarm = true)
+        private bool ChargeAccountAccessGas(ref long gasAvailable, EvmState vmState, Address address, IReleaseSpec spec, bool chargeForWarm = true)
         {
             // Console.WriteLine($"Accessing {address}");
             
             bool result = true;
             if (spec.UseHotAndColdStorage)
             {
+                if (_txTracer.IsTracingAccess) // when tracing access we want cost as if it was warmed up from access list
+                {
```

Comment:
why?...

Reply:
oh...

Reply:
this should be only in IsCold -> if Is Cold then report access and do not charge gas